answer/166ryoryo/JS/11/isSame.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+<meta charset="utf-8">
+<title></title>
+<meta name="keywords" content="">
+<meta name="description" content="">
+</head>
+
+<body>
+        <div class="main">
+          <input type="text" id="text1" placeholder="aiueo1">
+          <input type="text" id="text2" placeholder="aiueo2">
+          <input type="button" value="比較" onclick="hikaku()">
+        </div>
+        <script>
+          function hikaku() {
+            const text1 = document.getElementById('text1').value;
+            const text2 = document.getElementById('text2').value;
+            if (text1 && text2) {
```

Comment:
nits

```suggestion
            if (text1とtext2のどっちか(もしくは双方)が空の時) { return }

            if (text1 === text2) {
              alert('true');
            } else {
              alert('false');
            }
```

みたいに書くと `if`のネストが浅くなって個人的に好きでsu